armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -818,14 +818,10 @@ def _isFuelLocked(self, b):
         - This serves as an example to check for fuel/clad locking/interaction found in SFRs.
         - A more realistic/physical implementation is reserved for ARMI plugin(s).
         """
-        c = b.getChildrenWithFlags(Flags.FUEL)
-        if len(c) == 0:  # pylint: disable=no-else-raise
-            raise RuntimeError("No fuel component within {0}!".format(b))
-        elif len(c) > 1:
-            raise RuntimeError(
-                "Cannot have more than one fuel component within {0}!".format(b)
-            )
-        self._componentDeterminesBlockHeight[c[0]] = True
+        c = b.getComponent(Flags.FUEL)
+        if c is None:
+            raise RuntimeError(f"No fuel component within {b}!")
+        self._componentDeterminesBlockHeight[c] = True
```

Comment:
Actually, this is confusing to me, don't you need to use the `fuelCladLocked` parameter too?

Reply:
Discussed offline. This is good for as a base class to build on.